packages/tooltip/Tooltip.tsx

```diff
@@ -9,34 +9,43 @@ const TooltipTypes = {
   bottom: 'bottom',
 };
 
-type TooltipTypesType = typeof TooltipTypes;
+type TooltipTypesType = keyof typeof TooltipTypes;
 
 // public
-interface TooltipProps {
-  type?: keyof TooltipTypesType,
+type TooltipProps = {
+  type?: TooltipTypesType,
   label: string,
-}
+};
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'bl-tooltip': TooltipProps & Partial<HTMLElement>,
```

Comment:
ok it was late and I was tired :D so nevermind. Let's stick with `HTMLElement` for all intrinsic elements